### DISCUSSION TITLE
Søknadsdialog - introside

### DISCUSSION THREAD

**Discussion by sjur-gr** [navikt/Designsystemet#2919] [Mønster og maler]
[Introsiden er en mal på Aksel](https://aksel.nav.no/monster-maler/soknadsdialog/introside-for-soknadsdialoger). Det har vært et veldig bredt samarbeid med mange produktteam, YA, innholdsdesignere, Fyll ut og Aksel. I denne tråden inviterer vi til tilbakemelding og samarbeid. Vi ønsker å forbedre både malen og veiledningen.

**Comment by MBegnum** [comment]
Hei!

Vi velger å ikke bruke samme size på H1 og H2, dvs vi setter begge H2 ("Hei, XXX" og "Før du søker") til Medium siden H1 (Søknad om YTELSE") er Large. I malen er siste H2 Large.

Vi velger også å beholde brukers erklæring i komponenten Confirmation Panel fra Aksel, det er ikke gjort her.

Bekreftelseslinjen i brukers erklæring er noe ulik her og i standardmalene, vi har brukt den som er her.

Det mangler NN og EN oversettelser av standardtekster, inkludert brukers erklæring. Vi har oversatt til NN men avventer EN.
